src/bag/bag_loader.py

```diff
@@ -73,8 +74,28 @@ def unpack_zip(self, *, table_name: str, endpoint: str):
             )
         return path_target
 
+    def preprocess_csv_rows(self, reader: csv.DictReader, writer: csv.DictWriter):
+        seen = {}
+        for row in reader:
+            key = row['Identificatie']
+            if (key not in seen):
+                seen[key] = row
+            elif row['Volgnummer'] > seen[key]['Volgnummer']:
+                seen[key] = row
+        
+        filtered_rows = list(seen.values())
```

Comment:
Deze regel is overbodig. `seen.values()` is al een iterator waar je doorheen kan loopen met `for row in seen.values()`